services/explorer-indexer/.env.example

```diff
@@ -1,9 +1,12 @@
 # --- Port used by backend to run the server --- #
 SERVER_PORT=3040
-
+WS_SERVER_PORT=80
 # --- DATABASE --- #
 DB_HOST=localhost
-DB_PORT=5432
-DB_USERNAME=postgres
+DB_PORT=55000

```

Comment:
Shouldn't this be 5432? That`s the default port of postgres. If locally it is custom, we use than in the .env but in the .env.example we should keep the default one.